go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca
```

Comment:
Looks like there is a pretty recent release of this library: https://github.com/xlab/treeprint/releases/tag/v1.1.0. Can we use that?

Reply:
Thanks. Done.